README.md

```diff
@@ -408,9 +408,14 @@ Writing `print(vim.inspect(x))` every time you want to inspect the contents of a
 
 ```lua
 function _G.dump(...)
-    local objects = vim.tbl_map(vim.inspect, {...})
-    print(unpack(objects))
-    return ...
+  local objects, v = {}, nil
+  for i = 1, select('#', ...) do
+    v = select(i, ...)
```

Comment:
Since Lua creates a scope for each loop iteration, you can put `local v = ...` inside the loop

```suggestion
  local objects = {}
  for i = 1, select('#', ...) do
    local v = select(i, ...)
```

Reply:
I usually do that because of perceived overhead of defining local variables inside a loop. Having actually tested it, even if it is present it is very, very small. So sure, will change this.

Reply:
That makes sense. I rarely call this function with more than 3 arguments so the overhead seems acceptable.